core/templates/dev/head/pages/admin/admin.html

```diff
@@ -135,6 +135,11 @@
     <script src="/templates/dev/head/domain/user/UserInfoObjectFactory.js"></script>
     <script src="/templates/dev/head/domain/utilities/UrlInterpolationService.js"></script>
 
+    <script src="/templates/dev/head/pages/landing-page/topic-landing-page.module.js"></script>
+    <script src="/templates/dev/head/pages/landing-page/topic-landing-page.controller.js"></script>
```

Comment:
The name of the module file should be `landing-page.module.js` and similarly the controller. The main module/controller file name should match with the directory it is in. Or change the directory name, any of the method would do but please ensure the convention.

Reply:
ok

Reply:
done